python_cross_site_scripting/QGB_QPSU/HTML.py

```
@@ -0,0 +1,2126 @@
+	draw.text(((size - w) // 2, (size - h) // 2 - size // 18), #  18 让上下居中
+			  char, fill=ImageColor.getrgb(foreground), font=font)
+	
+	from qgb import pil;return pil.pil_image_to_bytes(img,'png') if background=='transparent' else pil.pil_image_to_bytes(img,'bmp')
+	return [[0 if pixel else 1 for pixel in row] for row in img.getdata()]# 转换为点阵数据
+bmp_char=get_bmp_char=get_char_img_bytes
+
+def get_bmp_bytes(rgb=None,size=(16,16)):
+	if not rgb:
+		rgb = U.get_or_set('get_bmp.rgb', (255,0,0))
+		size = U.get_or_set('get_bmp.size', 16)
+	if py.isint(size):size=[size,size]
```

Ruff (0.13.3)

Multiple statements on one line (colon)

(E701)